deploy/SharedService_crd.yaml

```diff
@@ -18,3 +18,12 @@ spec:
           properties:
             image:
               type: string
+            clusterServiceClassExternalName:
+              type: string
+            clusterServiceClassName:
```

Comment:
do we need both of these can one be found out from the other

Reply:
This is to allow the creator of this CRD to add a human-friendly external name, but that is slower to look up, so after the service class name is found it is stored in the CRD for future faster lookups.